setup.py

```diff
@@ -55,7 +55,7 @@ def get_tests_require() -> List[str]:
 def get_extras_require() -> Dict[str, List[str]]:
 
     requirements = {
-        "checking": ["black", "hacking", "isort", "mypy", "blackdoc"],
+        "checking": ["black", "hacking", "isort", "mypy==0.790", "blackdoc"],
```

Comment:
Not a strong opinion but leaving a TODO to remove the constraint or explain why it was introduced might help us coming back to it.

Reply:
Thanks! I added. PTAL.